ansible_collections/arista/avd/roles/eos_designs/templates/l3_edge/logic.j2

```diff
@@ -5,7 +5,10 @@
 {% set l3_edge_data.isis_interfaces = [] %}
 {% for p2p_link in l3_edge.p2p_links | arista.avd.natural_sort %}
 {%     if p2p_link.profile is arista.avd.defined and l3_edge.p2p_links_profiles is arista.avd.defined %}
-{%         set p2p_profile = l3_edge.p2p_links_profiles[p2p_link.profile] | arista.avd.default({}) %}
+{%         set p2p_profile = l3_edge.p2p_links_profiles | arista.avd.default() |
```

Comment:
When the variable is not defined, the default value is set to **None/null**. So performing the next sequence of operations over the null value is throwing the error.

Same for below.

Reply:
corrected, facing an error in CI